app/test_main.py

```diff
@@ -1,3 +1,26 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_correct_result_if_ages_bigger_then_14():
+    cat_age, dog_age = 28, 28
+    assert get_human_age(cat_age, dog_age) == [3, 2]
```

Comment:
```suggestion
    assert get_human_age(28, 28) == [3, 2]
```
Do not create extra variables